src/assets/styles/base.scss

```diff
@@ -286,3 +286,9 @@ ul.list {
 .invalid-section {
   border-left: 3px solid $app-red;
 }
+
+.default-text {
+  font-weight: normal;
+  color: $gray7;
+  font-size: 1rem;
+}
```

Comment:
I think you should consider overriding the default Vuetify text instead.

See what we did for NR here:
https://github.com/bcgov/namerequest/blob/9b3ff47e5f8b0a040b3764e6358603d06bed87fb/src/assets/scss/overrides.scss#L8

Rationale: you have defined a new class for something that already exists.